src/theory/arith/inference.h

```diff
@@ -9,13 +9,13 @@
  ** All rights reserved.  See the file COPYING in the top-level source
  ** directory for licensing information.\endverbatim
  **
- ** \brief Inference enumeration for non-linear arithmetic.
+ ** \brief Inference enumeration for arithmetic.
  **/
 
 #include "cvc4_private.h"
 
-#ifndef CVC4__THEORY__ARITH__NL__INFER_INFO_H
-#define CVC4__THEORY__ARITH__NL__INFER_INFO_H
+#ifndef CVC4__THEORY__ARITH__INFERENCE_H
+#define CVC4__THEORY__ARITH__INFERENCE_H
```

Comment:
Sorry this may be a bit of a pain, but I think we should update the IDs to have `NL_` to indicate they are coming from the non-linear solver. Also `T_` should become `NLT_`.

Reply:
Done.